circle.yml

```diff
@@ -23,12 +23,24 @@ deployment:
     commands:
       - docker login -e . -u $DOCKER_USER -p $DOCKER_PASS quay.io
       - docker tag tacstack quay.io/iseexchange/tacstack:latest
+      - docker tag mysqldatavol quay.io/iseexchange/tacstack:mysqldatavol
+      - docker tag ftpdatavol quay.io/iseexchange/tacstack:ftpdatavol
       - docker push quay.io/iseexchange/tacstack:latest
+      - docker push quay.io/iseexchange/tacstack:mysqldatavol
+      - docker push quay.io/iseexchange/tacstack:ftpdatavol
       - docker tag tacstack quay.io/iseexchange/tacstack:${CIRCLE_SHA1:0:7}
+      - docker tag mysqldatavol quay.io/iseexchange/tacstack:mysqldatavol_${CIRCLE_SHA1:0:7}
+      - docker tag ftpdatavol quay.io/iseexchange/tacstack:ftpdatavol_${CIRCLE_SHA1:0:7}
       - docker push quay.io/iseexchange/tacstack:${CIRCLE_SHA1:0:7}
+      - docker push quay.io/iseexchange/tacstack:mysqldatavol${CIRCLE_SHA1:0:7}
```

Comment:
here's why:

we're technically using access tokens to push, not a user.
the access token has push rights; the rest of us have read-only.

from a workflow perspective, this prevents a person from accidentally pushing directly.
all change goes through CI for audit purposes.

in an emergency, we can always change our access perms.